packages/manager/apps/telecom/src/app/telecom/telephony/line/assist/rma/rma.constants.js

```diff
@@ -0,0 +1,35 @@
+export const MAC_ADDRESS_FORMAT_REGEX = /\w{1,2}/g;
+
+export const RMA_URL_PREFIX =
+  'http://www.ovh.com/cgi-bin/telephony/rma.pl?reference=';
```

Comment:
**question:** Could you please check if you can use https. Also fixes the Security hotspot reported during SonarCloud analysis.
```suggestion
  'https://www.ovh.com/cgi-bin/telephony/rma.pl?reference=';
```

Reply:
It's work with https, I fixed.